shopinvader_customer_multi_user/tests/test_multi_user_service_partner_domain.py

```diff
@@ -2,283 +2,101 @@
 # @author Simone Orsi <simahawk@gmail.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from .common import TestMultiUserCommon
+from .common import TestMultiUserPartnerDomainCommon
 
 
-class TestMultiUserServicePartnerDomain(TestMultiUserCommon):
+class TestMultiUserServicePartnerDomain(TestMultiUserPartnerDomainCommon):
     """Test partner domains for services"""
```

Comment:
what to expect on this file:
- setup was moved to common.
- some helpers moved to common
- tests now use more common helper methods / recordset shortcuts
- tests cases remain the same, results remain the same